pytorch_toolkit/object_detection/requirements.txt

```diff
@@ -1,12 +1,13 @@
 cython==0.29.12
-numpy==1.16.4
-torch==1.1
-mmcv==0.2.10
-tensorboardX==1.8
-tensorboard==1.14.0
+matplotlib==3.1.1
+mmcv==0.2.16
+networkx==2.3
+numpy==1.17.5
 opencv-python==4.1.0.25
+ptflops==0.3
 pycocotools==2.0.0
+tensorboard==1.14.0
+tensorboardX==1.8
 terminaltables==3.1.0
-matplotlib==3.1.1
-ptflops==0.3
-networkx==2.3
+torch==1.4.0
+torchvision==0.5.0
```

Comment:
Maybe it would be better to use mmdetection's requirements explicitly? Or we need PyTorch and other packages anyway?